python/ray/experimental/workflow/storage/filesystem.py

```diff
@@ -257,3 +258,22 @@ async def get_step_status(self, workflow_id: str,
     @property
     def storage_url(self) -> str:
         return str(self._workflow_root_dir)
+
+    async def load_actor_class_body(self, workflow_id: str) -> type:
```

Comment:
I kind of feeling that our API for storage is becoming too long and the only different part actually is the KEY itself. I'm good for it right now, later we should have a refactor of this part so that we can reduce duplicate logic between different storages.

Reply:
Agree!